pkg/model/master_volumes.go

```diff
@@ -378,3 +382,24 @@ func (b *MasterVolumeBuilder) addAzureVolume(
 	}
 	c.AddTask(t)
 }
+
+func (b *MasterVolumeBuilder) addScalewayVolume(c *fi.ModelBuilderContext, name string, volumeSize int32, zone string, etcd kops.EtcdClusterSpec, m kops.EtcdMemberSpec, allMembers []string) {
+	// QUESTION : implement a SafeClusterName fct like DO and GCE ??
```

Comment:
What is the connexion between a Scaleway volume and a safe cluster name? I'm not sure to follow